cars_list.php

```diff
@@ -4,8 +4,8 @@
 include "head.php";
 include  "navigation.php";
 
+function allCars($conn) {
```

Comment:
Visibility MUST be declared on all methods.

The opening brace MUST go on its own line

https://www.php-fig.org/psr/psr-2/#43-methods